homeassistant/components/overkiz/water_heater_entities/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Water heater entities for the Overkiz (by Somfy) integration."""
```

Comment:
You need to exclude this from codecoverage in .coveragerc, or you would need to write tests.